all.sh

```diff
@@ -11,6 +11,25 @@ if [[ -z ${INPUT_TRUNK_TOKEN} ]]; then
     --output-file .trunk/landing-state.json \
     --github-commit "${GITHUB_SHA}" \
     ${INPUT_ARGUMENTS}
+elif [[ ${CHECK_ALL_MODE} == "hold-the-line" ]]; then
+  latest_raw_upload="$(mktemp)"
+  prev_ref=$("${TRUNK_PATH}" get-latest-raw-upload \
+    --series "${INPUT_UPLOAD_SERIES:-${GITHUB_REF_NAME}}" \
+    --token "${INPUT_TRUNK_TOKEN}" \
+    "${latest_raw_upload}") || prev_ref="NOT_FOUND"
```

Comment:
As I'm understanding this, `get-latest-raw-upload` outputs the previous git ref that was uploaded successfully? Are there any concerns about progress bars or other information, or is this guaranteed to only output that ref when EXIT_SUCCESS. Just want to sanity check.